src/main/java/athena/Storage.java

```diff
@@ -114,6 +115,8 @@ public TaskList loadTaskListData() throws StorageException {
                 throw new StorageLoadFailException();
             } catch (TaskDuringSleepTimeException e) {
                 throw new StorageCorruptedException(data);
+            } catch (TaskIsDoneException e) {
+                throw new StorageCorruptedException(data);
```

Comment:
Hmm why would this place throw this exception?

Reply:
Because line 104 uses `markTaskAsDone()` so I need to catch this exception otherwise there'll be error

Reply:
Ohh ok. I feel that we should have something like an `assert(false)` here because if the program does reach this line, it means we have done something wrong, since by right `TaskIsDoneException` shouldnt happen when loading from storage. But not really a big deal.

Reply:
Okay I've made the change